cmd/operator-sdk/build/cmd.go

```diff
@@ -126,20 +152,19 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 			log.Fatalf("Failed to build operator binary: %v", err)
 		}
 	}
+	doImageBuild("build/Dockerfile", image)
+	return nil
+}
 
-	image := args[0]
-
+// doImageBuild will execute the build command for the Dockerfile and image informed
+func doImageBuild(dockerFilePath, image string) {
```

Comment:
This should probably return an error, and we can bubble it up to `buildFunc`, where we can call `log.Fatal`